src/sorting.h

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file sorting.h
+ * prototypes for sorting.c
+ *
+ * @author  Ben Gardner
+ * @license GPL v2+
+ */
+#ifndef SORTING_H_INCLUDED
+#define SORTING_H_INCLUDED
+
+#include "uncrustify_types.h"
+
+
+void sort_imports(void);
```

Comment:
It is included as some types from `uncrustify_types.h`are used in `sorting.cpp`. However in `sorting.h` `uncrustify_types.h`was included already. So it was not necessary to include it again.
Thus I've remove the inclusion in `sorting.cpp`.

Regarding missing functions:  I' not sure which functions you are missing. Prototypes for static functions shall not go to the header but into the source file. Some static declarations are missing. I address that in change request #927. I have not seen missing prototypes for public functions.